make/targets_list.mk

```diff
@@ -40,7 +40,58 @@ UNSUPPORTED_TARGETS := \
     CC3D_OPBL \
     CJMCU \
     MICROSCISKY \
-    NAZE
+    NAZE \
+    AIORACERF3 \
+    AIR32 \
+    AIRHEROF3 \
+    ALIENFLIGHTF3 \
+    BEEBRAIN_V2D \
+    BEEBRAIN_V2F \
+    BEESTORM \
+    BETAFLIGHTF3 \
+    CHEBUZZF3 \
+    COLIBRI_RACE \
+    CRAZYBEEF3DX \
+    CRAZYBEEF3FR \
+    CRAZYBEEF3FS \
+    DOGE EACHIF3 \
+    FF_ACROWHOOPSP \
+    FF_KOMBINI \
+    FF_PIKOBLX \
+    FF_RADIANCE \
+    FLIP32F3OSD \
+    FRSKYF3 \
+    FURYF3 \
+    FURYF3OSD \
+    IMPULSERCF3 \
+    IRCFUSIONF3 \
+    IRCSYNERGYF3 \
+    ISHAPEDF3 \
+    KISSCC \
+    KISSFC \
+    LUMBAF3 \
+    LUXV2_RACE \
+    LUX_RACE \
+    MIDELICF3 \
+    MOTOLAB \
+    MULTIFLITEPICO \
+    OMNIBUS \
+    RACEBASE \
+    RCEXPLORERF3 \
+    RG_SSD_F3 RMDO \
```

Comment:
You've got one target per line, except for RG_SSD_F3 and RMDO, those are on one line right now. Am I allowed to beg for consistency?

Reply:
Yes, you are - well spotted.

As usual, you are allowed to keep all the bug you find in your review. :wink: 

Fixed.